.travis.yml

```diff
@@ -1,12 +1,14 @@
 language: ruby
 before_install:
-  - rm ./Gemfile.lock
+  - rm ./gemfiles/Gemfile.*.lock
 cache: bundler
 rvm:
-  - 2.4.1
   - 2.5
```

Comment:
2.5 is soon EOL, so I'm not opposed to removing that one too (see https://www.ruby-lang.org/en/downloads/). We can wait until it's actually EOL though.

Reply:
Yeah, let's do this separately.